lib/controllers/base_controller.rb

```diff
@@ -50,5 +50,20 @@ def prevent_unauthorized_modifications
         redirect back
       end
     end
+
+    def javascript_imports
+      imports =
+        {
+          imports: {
+            "@hotwired/turbo": "https://cdn.jsdelivr.net/npm/@hotwired/turbo@8.0.5/+esm",
+            "@hotwired/stimulus": "https://cdn.jsdelivr.net/npm/@hotwired/stimulus@3.2.2/+esm",
```

Comment:
We want to keep the ability to work on the wiki without internet access, so this needs to be different. Can we utilise the gem https://github.com/hotwired/turbo-rails? That depends on [`actionpack`](https://rubygems.org/gems/actionpack) and then we get a bunch of things, like `activesupport` 😞